integration-test/Css.ts

```diff
@@ -164,9 +173,8 @@ class CssBuilder<T extends Properties1> {
 
   // float
   get fl() { return this.add("float", "left"); }
-  get fn() { return this.add("float", "none"); }
   get fr() { return this.add("float", "right"); }
-  f(value: Properties["float"]) { return this.add("float", value); }
+  float(value: Properties["float"]) { return this.add("float", value); }
```

Comment:
I removed `fn` and changed the old `f`-that-was-`float` to just `float` b/c who uses `float` anymore? :shrug: